packages/op-app/src/test-utils/react.ts

```diff
@@ -0,0 +1,88 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
+import {
+  renderHook as rtl_renderHook,
+  type RenderHookOptions,
+  type RenderHookResult,
+  waitFor as rtl_waitFor,
+  type waitForOptions,
+} from '@testing-library/react'
+import { createElement } from 'react'
+import { createConfig, http, WagmiProvider } from 'wagmi'
+import { connect, disconnect } from 'wagmi/actions'
+import { mock } from 'wagmi/connectors'
+
+import { l1, l2 } from './chains'
+import { TEST_ACCOUNT } from './constants'
+
+export { act, cleanup } from '@testing-library/react'
+
+export const opConfig = createConfig({
+  chains: [l1, l2],
+  connectors: [
+    mock({ accounts: [TEST_ACCOUNT] }),
+    mock({ accounts: [TEST_ACCOUNT] }),
+  ],
+  pollingInterval: 100,
+  storage: null,
+  transports: {
+    [l1.id]: http(),
+    [l2.id]: http(),
+  },
+})
+
+// needed to preserve WagmiConfig prototype
```

Comment:
when you deconstruct an object its functions on the prototype are not copied over